Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "AWSS3"
 uuid = "1c724243-ef5b-51ab-93f4-b0a88ac62a95"
-version = "0.8.8"
+version = "0.8.9"
```

Comment:
This seems like a breaking change?

Reply:
After the rest of my changes, yes this is unambiguously breaking.